index.rst

```diff
@@ -61,6 +61,7 @@ You may also directly use related links to see documents which match you the mos
    :hidden:
 
    EPICS Website <https://epics-controls.org>
+   EPICS 7.0 Release Notes <https://docs.epics-controls.org/projects/base/en/latest/RELEASE_NOTES.html>
```

Comment:
I wouldn't personally have put the Release Notes link up here, maybe it belongs under "Installation"? Preferably somewhere to it will appear in the body of the page as well as in the side-bar.